lms/security.py

```diff
@@ -117,7 +116,7 @@ def get_policy(request: Request):
 
         if path in {"/lti_launches", "/content_item_selection"}:
             # Actual LTI backed authentication
-            return LTIUserSecurityPolicy(get_lti_user_from_launch_params)
+            return LaunchParamsLTIUserPolicy()
```

Comment:
Now each of this branches returns a policy object which is built the same way.

Reply:
Strategy pattern FTW